src/components/Graph/DefGrad.js

```diff
@@ -12,7 +12,14 @@ export default class DefGraph extends PureComponent<Props> {
   render() {
     const { height, color } = this.props;
     return (
-      <LinearGradient id="grad" x1="0" y1="0" x2="0" y2={height}>
+      <LinearGradient
+        id="grad"
```

Comment:
Are id globals?

Reply:
I could be wrong but I think the defs are global, hence the fix with unique ids we had to do elsewhere. However since nowhere in the app do we render more than one graph in the same screen it should be fine (?) 

This DefGrad essentially is a def used in the graph, worst case scenario it changes the gradient a bit. We can add a unique identifier for peace of mind later on. https://github.com/LedgerHQ/ledger-live-mobile/blob/d374d00b84465c040c9ce6a071b1ebd3129a071a/src/components/Graph/index.js#L82-L88

Reply:
Portfolio tab and account tab can be simultaneously rendered to me (as tabs content don't get unmounted)